### PR TITLE
cleanup QTimer management in ThreadedAssignment

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 49ea560e4e2fec110bb4c2e29376a4fc1b4aae21 available at: https://gist.github.com/94d95885b356b691a41e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6718.dmg

**Comment by jherico** [comment]
I don't see where the `_domainServerTimer` is started.

**Comment by birarda** [comment]
@jherico good catch - I missed that since the node was able to stay connected to the domain with its stats packets. Fixed.

**Comment by hifi-gustavo** [comment]
Build console output for commit: a0d6001fa506c1cd78113befab5e6f08e728da10 available at: https://gist.github.com/94d95885b356b691a41e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6718.dmg

File: libraries/networking/src/ThreadedAssignment.cpp [hunk 1]
```diff
@@ -111,28 +102,12 @@ void ThreadedAssignment::sendStatsPacket() {
...
 
-void ThreadedAssignment::startSendingStats() {
-    // send the stats packet every 1s
-    if (!_statsTimer) {
-        _statsTimer = new QTimer;
-        connect(_statsTimer, &QTimer::timeout, this, &ThreadedAssignment::sendStatsPacket);
-    }
-    
-    _statsTimer->start(1000);
-}
```
Comment (AndrewMeadows) [inline libraries/networking/src/ThreadedAssignment.cpp]: Similarly for **_statsTimer** the old **start()** was lost and there is no new one.

Comment (birarda) [inline libraries/networking/src/ThreadedAssignment.cpp]: This is what starts it - from `commonInit` above.

```
// start sending stats packet once we connect to the domain
connect(&nodeList->getDomainHandler(), SIGNAL(connectedToDomain(const QString&)), &_statsTimer, SLOT(start()));
```

**Comment by hifi-gustavo** [comment]
Build console output for commit: 02669ce322d8e787887f78fd8c7449849cbc44ad available at: https://gist.github.com/94d95885b356b691a41e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6718.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ddb8b6726703588b4a2f5aba053bbfea58e64ce2 available at: https://gist.github.com/94d95885b356b691a41e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6718.dmg

**Comment by AndrewMeadows** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: ddb8b6726703588b4a2f5aba053bbfea58e64ce2 available at: https://gist.github.com/94d95885b356b691a41e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6718.dmg

**Comment by AndrewMeadows** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: ddb8b6726703588b4a2f5aba053bbfea58e64ce2 available at: https://gist.github.com/94d95885b356b691a41e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6718.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6718.dmg

**Pull request merged**